src/sentry/debug_files/artifact_bundles.py

```diff
@@ -195,6 +195,7 @@ def index_urls_in_bundle(
 
 @sentry_sdk.tracing.trace
 def maybe_renew_artifact_bundles_from_processing(project_id: int, used_download_ids: list[str]):
+    # TODO this rollout comparison is backwards
```

Comment:
this is intentional because its an early-return, plus I should take another look if this code can be salvaged and activated at some point.